librz/core/cmd_type.c

```diff
@@ -1883,6 +1919,84 @@ RZ_IPI RzCmdStatus rz_type_define_handler(RzCore *core, int argc, const char **a
 	return RZ_CMD_STATUS_OK;
 }
 
+RZ_IPI RzCmdStatus rz_type_list_enum_handler(RzCore *core, int argc, const char **argv, RzOutputMode mode) {
+	const char *enum_name = argc > 1 ? argv[1] : NULL;
+	// TODO: Reconsider the `te <enum_name> <member_value>` syntax change
```

Comment:
This is something that always bugs me... Having the same handler that does both listing and addition. I'm still not sure if it's a real problem or not and, if it is, how to improve the situation... Adding yet another command is a bit meh.

Reply:
It doesn't add the value, it can only be done through `td` or manually in SDB. But it gives fine grained access for the enum members.